libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -50,8 +50,14 @@ const readingListReducer = createReducer(
   on(ReadingListActions.addToReadingList, (state, action) =>
     readingListAdapter.addOne({ bookId: action.book.id, ...action.book }, state)
   ),
+  on(ReadingListActions.confirmedAddToReadingList, (state, action) =>
```

Comment:
This is supposed to be an optimistic update, the confirm action shouldn't do anything at all.